code/modules/recycling/conveyor2.dm

```diff
@@ -394,10 +396,10 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	desc = "A conveyor belt assembly."
 	icon = 'icons/obj/recycling.dmi'
 	icon_state = "conveyor_construct"
-	max_amount = 30
+	max_amount = 100
 	singular_name = "conveyor belt"
-	w_class = WEIGHT_CLASS_BULKY
-	materials = list(/datum/material/iron = 3000)
+	w_class = WEIGHT_CLASS_NORMAL
+	materials = list(/datum/material/iron = 500)
```

Comment:
```suggestion
	materials = list(/datum/material/iron = 1000)
```

Reply:
this one has to be here, currently these things munch on the station's iron supply way too hard